src/ConfigInterface.php

```diff
@@ -97,6 +99,11 @@ public function getRules(): array;
      */
     public function getUsingCache(): bool;
 
+    /**
+     * Returns true if execution should fail on unsupported PHP version.
+     */
+    public function getFailOnUnsupportedVersion(): bool;
```

Comment:
(not to solve before concluding [this](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8733/files#r2145614223))
BC break, sth we do not accept currently

Reply:
How does this project then add new config in a minor? Let me know the correct way (example) to do it and I'll fix it.

Reply:
Similarly to https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.75.0/src/ParallelAwareConfigInterface.php

Reply:
Thanks, I'll fix it like that.